streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/transfer/DataReader.java

```diff
@@ -38,26 +44,25 @@ public DataReader(List<String> inputChannels,
       seqIds[i] = 0;
       msgIds[i] = 0;
     }
-    long timerInterval = Long.parseLong(
-        conf.getOrDefault(Config.TIMER_INTERVAL_MS, "-1"));
-    String channelType = conf.get(Config.CHANNEL_TYPE);
+    long timerInterval = workerConfig.transferConfig.readerTimerIntervalMs();
+    TransferChannelType channelType = workerConfig.transferConfig.channelType();
     boolean isMock = false;
-    if (Config.MEMORY_CHANNEL.equals(channelType)) {
+    if (TransferChannelType.MEMORY_CHANNEL == channelType) {
       isMock = true;
     }
-    boolean isRecreate = Boolean.parseBoolean(
-        conf.getOrDefault(Config.IS_RECREATE, "false"));
+    boolean isRecreate = workerConfig.transferConfig.readerIsRecreate();
+
     this.nativeReaderPtr = createDataReaderNative(
         initialParameters,
         inputChannelsBytes,
         seqIds,
         msgIds,
         timerInterval,
         isRecreate,
-        ChannelUtils.toNativeConf(conf),
+        ChannelUtils.toNativeConf(workerConfig),
         isMock
     );
-    LOGGER.info("create DataReader succeed");
+    LOGGER.debug("Create DataReader succeeded.");
```

Comment:
Why change from `info` to `debug`?  This log can be used to confirm data reader is created successfully

Reply:
But in fact, u can not  figure out which worker the reader or writer belongs to if u don't logging the name of the worker(or vertex).

Reply:
LOGGER.info("Create DataWriter succeed for worker: {}.", workerConfig.workerInternalConfig.workerName());